File: lc771.java
public class lc771 {
    // 771. Jewels and Stones
    public static void main(String[] args) {
        String jewels = "aA", stones = "aAAbbbb";
        int ans=0;
        for (int i=0;i<stones.length();i++){
            String s= String.valueOf(stones.charAt(i));
            if (jewels.contains(s)){
                ans++;
            }
        }
        System.out.println(ans);
    }
}

File: lc771.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        jewelsSet = set(jewels)
        return sum(stone in jewelsSet for stone in stones)
print(Solution.numJewelsInStones(0,jewels = "aA", stones = "aAAbbbb"))